tests/ggsmark.spec.js

```diff
@@ -52,4 +52,15 @@ new line
     // Assert
     expect(result).toMatchSnapshot()
   })
+  test('text-center', () => {
+    let string = ':text-center Test123'
+    let result = ggsmark(string)
+
+    expect(result).toContain(`
+<p>
+</p><div style=\"text-allign: center\" class=\"text-center\">  Test123
+</div>
+<p></p>`)
```

Comment:
```suggestion
  expect(result).toContain(`<p>
</p><div style=\"text-allign: center\" class=\"text-center\">  Test123
</div>
<p></p>`)
```
Minor change to this to make it work